lib/modules/platform/bitbucket/schema.ts

```diff
@@ -20,3 +20,38 @@ const PagedSchema = z.object({
 export const PagedSourceResultsSchema = PagedSchema.extend({
   values: z.array(SourceResultsSchema),
 });
+
+export const RepoInfo = z
+  .object({
+    parent: z.unknown().optional(),
+    owner: z.object({
+      username: z.string(),
+    }),
+    mainbranch: z.object({
+      name: z.string(),
+    }),
+    has_issues: z.boolean(),
+    uuid: z.string(),
+    full_name: z.string(),
+    is_private: z.boolean(),
```

Comment:
have you validated that all those fields are always returned from API?

Reply:
I think it's pretty safe to go with premise they all are, and correct our assumption later

Reply:
Actually yes, seems like `owner` field doesn't have `username` for all types of token, e.g. my local testing repo token

Reply:
I believe they're all important, at least I can't confidently tell it's okay to ignore any them

Reply:
We should be more defensiv, otherwise we break existing users.

You can validate in transform to have valies and log warnings if not, so we can trim later instead of break users.

There is or will be a bitbucket hosted app soon, so we really shouldn't break that!